java/kotlin-extractor/src/main/kotlin/KotlinUsesExtractor.kt

```diff
@@ -9,12 +9,14 @@ import org.jetbrains.kotlin.backend.common.lower.parentsWithSelf
 import org.jetbrains.kotlin.backend.jvm.ir.propertyIfAccessor
 import org.jetbrains.kotlin.builtins.StandardNames
 import org.jetbrains.kotlin.descriptors.*
+import org.jetbrains.kotlin.ir.ObsoleteDescriptorBasedAPI
```

Comment:
I assume that you can't see an easy way to do this without the obsolete API?

Reply:
I'm afraid not; their map of methods that get special generics treatment was keyed by a descriptor. If the obsolete API goes away we can always just copy-paste the thing in in a slightly different form since it's unlikely to change over time.